projects/Benchmarks/WireFormatting/WireFormatting_Read_BasicAck.cs

```diff
@@ -16,9 +16,9 @@ public WireFormatting_Read_BasicAck()
         }
 
         [Benchmark(Baseline = true)]
-        public object ReadFromSpan()
+        public void ReadFromSpan()
```

Comment:
This might not have the behavior you want. The compiler could optimize away the logic since a new variable is being created without being used. Having the method return the new object it will force it to stay. See here: https://benchmarkdotnet.org/articles/guides/good-practices.html#avoid-dead-code-elimination

Reply:
I was thinking about it as well, the benchmark didn't change though.
Why did I change it? object return would box and I can't put the actual struct there, as it is internal. I haven't found a way to declare benchmarks as internal and being able to execute it. Is there?